packages/eslint-plugin-wpcalypso/lib/rules/i18n-unlocalized-url.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * @file Disallow using unlocalized URL strings
+ * @author Automattic
+ * @copyright 2023 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+
+//------------------------------------------------------------------------------
+// Constants
+//------------------------------------------------------------------------------
+
+const LOCALIZABLE_URLS = [
+	// apps.wordpress.com/**
+	/^(https?:)?\/\/apps\.wordpress\.com/i,
+
+	// automattic.com/cookies/
+	/^(https?:)?\/\/automattic\.com\/cookies\/?$/i,
+
+	// automattic.com/privacy/
+	/^(https?:)?\/\/automattic\.com\/privacy\/?$/i,
+
+	// wordpress.com/tos/
+	/^(https?:)?\/\/wordpress\.com\/tos\/?$/i,
+
+	// wordpress.com/blog/
+	/^(https?:)?\/\/wordpress\.com\/blog\/?$/i,
+
+	// wordpress.com/forums/
+	/^(https?:)?\/\/wordpress\.com\/forums\/?$/i,
```

Comment:
I guess we intentionally don't expect anyone to link to a particular forum thread? Perhaps we should envision that and add support for it?

Reply:
Forum threads only exist in the specific forums site (i.e. language) they were created in. Therefore, we only want to localize the root URL of the forums sites.